services/app-web/src/index.tsx

```diff
@@ -57,17 +57,25 @@ Amplify.configure({
 
 const authMode = process.env.REACT_APP_AUTH_MODE
 assertIsAuthMode(authMode)
+const cache = new InMemoryCache()
+const defaultOptions: DefaultOptions = {
+    watchQuery: {
+      fetchPolicy: 'network-only',
+      errorPolicy: 'ignore',
```

Comment:
Pearl - can we remove off any `errorPolicy` changes here and just scope this to `fetchPolicy`. Sorry I missed this. cc: @macrael